clap_generate/src/generators/mod.rs

```diff
@@ -217,9 +235,13 @@ mod tests {
     fn common() -> App<'static> {
         let mut app = App::new("myapp")
             .subcommand(
-                App::new("test")
-                    .subcommand(App::new("config"))
-                    .arg(Arg::new("file").short('f').long("file")),
+                App::new("test").subcommand(App::new("config")).arg(
+                    Arg::new("file")
+                        .short('f')
+                        .short_alias('h')
```

Comment:
You might want to use something that is not `h` since it will clash with `help` short.

Reply:
Oh right sorry